packages/react/src/primitives/shared/utils.ts

```diff
@@ -7,40 +7,88 @@ import postcssJs from 'postcss-js';
 import { customAlphabet } from 'nanoid/non-secure';
 const nanoid = customAlphabet('1234567890abcdef', 12);
 
+import { theme } from '@aws-amplify/ui';
+
 import {
   ComponentPropsToStylePropsMap,
   ComponentPropToStyleProp,
   ViewProps,
 } from '../types/index';
 
+import { getValueAtCurrentBreakpoint } from './responsive/utils';
+import { useBreakpoint } from './responsive/useBreakpoint';
+import { Breakpoint, Breakpoints } from '../types/responsive';
+
 export const prefixer = postcssJs.sync([autoprefixer]);
 
+const {
+  values: breakpoints,
```

Comment:
@zchenwei @dbanksdesign Is there a better way to consume the breakpoints? I was hesitant to use the `useTheming` hook since I think that would require customers to use the Provider?

Reply:
That's a valid point. But if we consume the theme(default) from ui package, how do we get the updated breakpoints if users override it? Do you handle this in your code?

Reply:
Oh nice! I'll give that a try, thanks!